content/registry/brave-tracer-java.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Brave
+registryType: tracer
+tags:
+    - java
+    - zipkin
+    - brave
+    - spring-cloud-sleuth
+repo: https://github.com/openzipkin/brave-opentracing
+license: Apache
+description: A friendly description for what my project does.
```

Comment:
Could you make the description field here more descriptive? It's what gets surfaced on the search result page.

Reply:
Yup, missed that one on the copy paste